library/ecp_invasive.h

```diff
@@ -40,8 +40,33 @@ typedef enum {
     MBEDTLS_ECP_MOD_SCALAR
 } mbedtls_ecp_modulus_type;
 
+/* Requred macros for ECP split.
+ * If MBEDTLS_ECP_WITH_MPI_UINT is defined the new bignum interface is used.
+ */
+#ifndef MBEDTLS_ECP_WITH_MPI_UINT
+
+/* Because of `check_names.py` a define is needed for every macro. */
+//#define MBEDTLS_ECP_WITH_MPI_UINT
```

Comment:
```suggestion

/* Uncomment this to enable the new bignum interface. */
//#define MBEDTLS_ECP_WITH_MPI_UINT

/* Requred macros for ECP split.
 * If MBEDTLS_ECP_WITH_MPI_UINT is defined the new bignum interface is used.
 */
#ifndef MBEDTLS_ECP_WITH_MPI_UINT
```

This is more what I meant before, then there is an obvious place to switch it on and off kind of thing.

Reply:
The commented out define never must be uncommented. It will build both ecp files and will cause building errors.

The new bignum interface can be enabled by passing `-DMBEDTLS_ECP_WITH_MPI_UINT` to make.

Reply:
It won't build both modules. If you notice the change above, the `#define MBEDTLS_ECP_WITH_MPI_UINT` line has been moved outside of the `#ifndef MBEDTLS_ECP_WITH_MPI_UINT` block. Both ecp modules include ecp_invasive.h (and before their respective include guards), and thus only one will be built.